apps/vendor-portal/.env

```diff
@@ -1,4 +1,3 @@
 APP_LOG_ENABLED=true
 APP_LOG_LEVEL=debug
-APP_PLATFORM_URL=https://vcmarketplace-platform.dev.govirto.com/
-
+APP_PLATFORM_URL=http://localhost:10645/
```

Comment:
Please, avoid comitting local variables. You can use `.env.local` file instead, it will be gitignored by default.